tests/test_metrics/test_losses.py

```diff
@@ -30,6 +30,37 @@ def test_ce_loss():
     loss_cls = build_loss(loss_cls_cfg)
     assert torch.allclose(loss_cls(fake_pred, fake_label), torch.tensor(200.))
 
+    # test bce_loss
+    cls_score = torch.Tensor([[-200, 100], [500, -1000], [300, -300]])
```

Comment:
`bce_loss` now only supports the input tensor with shape (n, 1).

Reply:
[yolov3 cls head](https://github.com/open-mmlab/mmdetection/blob/5c7c74d77dfba3663431fe00b6872dfff8bcce1d/configs/yolo/yolov3_d53_mstrain-608_273e_coco.py#L29) uses bce loss, it's [pred input tensor](https://github.com/open-mmlab/mmdetection/blob/5c7c74d77dfba3663431fe00b6872dfff8bcce1d/mmdet/models/dense_heads/yolo_head.py#L494) has  shpae (batchsize, number_booxes, number_class)